fix: bump x-crypto@v0.31.0 (#903)
kyma-project/cloud-manager: docs/contributor/dependency-graph.md

# Dependency Graph

> Sometimes it is necessary to investigate dependency graph and find out dependency paths to certain package.

Reviewer: ```suggestion
Sometimes, it is necessary to investigate dependency graphs and find the paths to a certain package.
```

This is especially helpful when resolving security CVE issues. This document documents a way to find necessary
information easily.

> ## Deptree

Reviewer: ```suggestion
## deptree
```

> Install [deptree](https://github.com/vc60er/deptree) in the GOBIN dir on the path.

Reviewer: ```suggestion
Install [deptree](https://github.com/vc60er/deptree) in the `GOBIN` dir on the path.
```

> Since it takes a lot of time, it will save time to record module list once, so deptree can be run quickly

Reviewer: ```suggestion
Since it takes a lot of time, record the module list once so that deptree can be run quickly,
```
